lib/fakeredis/zset.rb

```diff
@@ -5,6 +5,10 @@ def []=(key, val)
       super(key, _floatify(val))
     end
 
+    def identical_scores?
+      values.all? { |key| key == values[0] }
```

Comment:
I was interested, so [quickly benchmarked this](https://gist.github.com/caius/7e107b51aa953b2cae03) and it looks like `values.uniq.size == 1` is faster than comparing each element in ruby. (Probably because `Array#uniq` is implemented in C in the runtime.)